nodejs18.x/full-stack/{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,176 @@
+# {{cookiecutter.project_name}}
```

Comment:
Can we add some explanation about which endpoints does this application have and how it works? Some sort of high level architecture diagram here so that when people instantiate this application they can see each component here.

Reply:
Please see added diagrams that further describe what is happening